Classes/Outlet/Pipe/ViewAwarePipeTrait.php

```diff
@@ -0,0 +1,31 @@
+<?php
+namespace FluidTYPO3\Flux\Outlet\Pipe;
+
+/*
+ * This file is part of the FluidTYPO3/Flux project under GPLv2 or later.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.md file that was distributed with this source code.
+ */
+
+use TYPO3\CMS\Extbase\Mvc\View\ViewInterface;
+
+/**
+ * trait to make a pipe view aware. must be paired with the ViewAwarePipeInterface to be recognised
```

Comment:
No. This trait satisfies `ViewAwarePipeInterface` instead of the other way around